reference_data/management/commands/update_gene_shet.py

```diff
@@ -0,0 +1,23 @@
+import logging
+from reference_data.management.commands.utils.update_utils import GeneCommand, ReferenceDataHandler
+from reference_data.models import GeneShet
+
+logger = logging.getLogger(__name__)
+
+
+class ShetReferenceDataHandler(ReferenceDataHandler):
+
+    model_cls = GeneShet
+    url = 'https://storage.googleapis.com/seqr-reference-data/Shet/Shet_Zeng_2023.tsv'
+
+    @staticmethod
+    def parse_record(record):
+        yield {
+            'gene_id': record['ensg'],
+            'shet': float(record['post_mean_shet']),
+            'shet_constrained': bool(int(record['shet_constrained'])),
```

Comment:
I'm not sure we need this field - we should load the Shet score for any gene we have data for, and we will define a cutoff in seqr for  whether or not to flag a gene based on the score cutoff, not a database field. What in the ticket do you anticipate will require this column?

Reply:
Hmm, the current cutoff looks like 0.1. The requirement ticket doesn't tell the cutoff explicitly. The new `'LoF constr' tag` will be displayed when anyone of LoF, HI, or Shet (this column is true).

Reply:
the cutoff is something that we use for display purposes. It should not live in the database in any way, so we should not be loading this